platforms/Linux/RPM/Fedora/34/swiftlang.spec

```diff
@@ -1,41 +1,61 @@
 %global debug_package %{nil}
 %global linux_version fedora
-%global swifttag 5.5.3-RELEASE
-%global swiftbuild swift-source
+%global swift_version 5.6-RELEASE
```

Comment:
should we name these 5.6.0? this could be better IMO since otherwise people will install 5.6 and not get updates?

Reply:
For Docker we are already tagging 5.6.0, we can do the same here.

Reply:
I will update `%global package_version   5.6` to `%global package_version   5.6.0` and in the future we can update the swift tag scheme to include `.0`

Reply:
Updated to use 5.6.0 package_version.